scripts/semantic-commits/get-binary-release-data.js

```diff
@@ -70,7 +70,7 @@ const getReleaseData = async (latestReleaseInfo) => {
       return
     }
 
-    const { data: pullRequest } = await octokit.request('GET /repos/{owner}/{repo}/pull/{pull_number}', {
+    const { data: pullRequest } = await octokit.request('GET /repos/{owner}/{repo}/pulls/{pull_number}', {
```

Comment:
so octokit does `pulls` when the conventional url is `pull`?

Reply:
yeah appear so: https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#get-a-pull-request
made the mistake of updating to `pull` which broken this in develop.